### PR TITLE
Feat: Add SolApi.token.getTokenPrice 

### PR DESCRIPTION
**Pull request by 3scava1i3r**
---
name: 'Add SolApi.token.getTokenPrice'
about: the 
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- x] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

We have the solApi endpoint with this as an option and still did not have the same in SDK

### Solution Description

Added getPrice endpoint support under Solana API in JS-SDK

### CODE REVIEW

File: packages/solApi/tsconfig.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
     "declarationDir": "./lib/",
     "rootDir": "./src"
   },
-  "include": ["src/**/*", "types/**/*"]
+  "include": ["src/**/*", "types/**/*", "getTokenPrice.ts"]
```
Comment (sogunshola) [inline packages/solApi/tsconfig.json:8]: Why do we have this file name included separately?

File: packages/solApi/src/resolvers/token/getTokenPrice.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    resultToJson: (data) => {
+      return {
+        nativePrice: {
+          value: data.nativePrice.value,
+          decimals: data.nativePrice.decimals,
+          name: data.nativePrice.name,
+          symbol : data.nativePrice.symbol
+        },
+        usdPrice: data.usdPrice,
+        exchangeAddress: data.exchangeAddress,
```
Comment (sogunshola) [inline packages/solApi/src/resolvers/token/getTokenPrice.ts:54]: This should be casted to SolAddress datatype, check implementation here https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/7a8d21de93e5ce381e6ef4046d08e70feb7390ca/packages/solApi/src/resolvers/account/getNFTs.ts#L33

File: packages/solApi/src/resolvers/token/getTokenPrice.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    apiToResult: (data: ApiResult) => {
+      return {
+        nativePrice: {
+          value: data.nativePrice.value,
+          decimals: data.nativePrice.decimals,
+          name: data.nativePrice.name,
+          symbol : data.nativePrice.symbol
+        },
+        usdPrice: data.usdPrice,
+        exchangeAddress: data.exchangeAddress,
```
Comment (sogunshola) [inline packages/solApi/src/resolvers/token/getTokenPrice.ts:41]: This should be casted to SolAddress datatype, check implementation here https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/7a8d21de93e5ce381e6ef4046d08e70feb7390ca/packages/solApi/src/resolvers/account/getNFTs.ts#L33

File: packages/solApi/src/resolvers/token/getTokenPrice.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        exchangeName: data.exchangeName
+      };
+    },
+    resultToJson: (data) => {
+      return {
+        nativePrice: {
+          value: data.nativePrice.value,
```
Comment (sogunshola) [inline packages/solApi/src/resolvers/token/getTokenPrice.ts:48]: This also should be casted to SolNative datatype. 

see the implementation here https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/7a8d21de93e5ce381e6ef4046d08e70feb7390ca/packages/solApi/src/resolvers/account/getSPL.ts#L35

**Review by sogunshola — CHANGES_REQUESTED** [review]
Left a few comments

File: packages/solApi/tsconfig.json (hunk 1 above)
Comment (3scava1i3r) [reply packages/solApi/tsconfig.json:8]: must be a minute error

File: packages/solApi/src/resolvers/token/getTokenPrice.ts (hunk 4 above)
Comment (3scava1i3r) [reply packages/solApi/src/resolvers/token/getTokenPrice.ts]: changes made

File: packages/solApi/src/resolvers/token/getTokenPrice.ts (hunk 3 above)
Comment (3scava1i3r) [reply packages/solApi/src/resolvers/token/getTokenPrice.ts]: changes made

**Review by sogunshola — COMMENTED** [review]
LGTM, @ErnoW please take a look when you have a moment

**Review by sogunshola — COMMENTED** [review]
please run `yarn format` so that prettier error will be corrected.

**Review by sogunshola — APPROVED** [review]

**Pull request merged**